tools/migrations/add_img_url_to_article.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE
```

Comment:
let's please prefix the name of the migrations with <year>-<month> as in the other examples in the folder.